templates/activities/_modal_image.html

```diff
@@ -0,0 +1,34 @@
+<script type="text/hyperscript">
+    def imageviewer.show(source)
+        set source_url to (<img /> in source) @data-original-url
+        set source_alt to (<img /> in source) @alt
+        set image to <#image-viewer img />
+        set figcaption to <#image-viewer figcaption />
+
+        -- fill information
+        set image@src to source_url
+        set image@alt to source_alt
+        set figcaption's textContent to source_alt
+
+        -- show image viewer
+        show #image-viewer with display:flex
+        transition #image-viewer's opacity from 0 to 1 over 300ms
+        js document.querySelector('#image-viewer button').focus()
```

Comment:
Without `focus()` close button cannot be focused using a keyboard. Also, I couldn't find a working hyperscript.

Reply:
OK, the hyperscript issue can be fixed by `focus() on first <#image-viewer button />`. What I need was an additional `first`.